builds/jenkins-build-pack-function/Jenkinsfile

```diff
@@ -848,7 +848,7 @@ def loadServerlessConfig(String runtime, def isEventSchdld, def isScheduleEnable
     sh "cp _config/aws-events-policies/custom-policy.yml ./policyFile.yml"
   }
 
-  if(!isEventSchdld) {
+  if((!isEventSchdld && ( configLoader.TVAULT && !configLoader.TVAULT.IS_ENABLED )) || config['domain'] == 'jazz' )) {
```

Comment:
What's the purpose of checking this flag: TVAULT.IS_ENABLED here?

Reply:
Same as above comment, currently we need the events/resources only when event enabled/vault enabled